### PR TITLE
Create/Update/Delete depends on registry flag

### PR DESCRIPTION
**Pull request by manodnyab**
## Description

Create/Update/Delete actions are now behind the registry flag

 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -255,6 +255,7 @@ with what features/services are supported.
         <registryKey key="aws.feature.ecsCloudDebug" description="Enables the Cloud Debug for ECS set of features" restartRequired="false" defaultValue="true"/>
         <registryKey key="aws.feature.ecsExec" description="Enables the features and actions associated with ECS Execute Command" restartRequired="false" defaultValue="false"/>
         <registryKey key="aws.feature.dynamoDb" description="Enables the features and actions associated with DynamoDB" restartRequired="false" defaultValue="true"/>
+        <registryKey key="aws.feature.moreResources.mutation" description="Enables the Create/ Update/ Delete features and actions associated with More resources" restartRequired="false" defaultValue="false"/>
```
Comment (kiiadi) [inline jetbrains-core/resources/META-INF/plugin.xml:258]: nit: spacing with the `/` in the description looks weird either do "create/update/delete" or "create / update / delete" is "More resources" a proper-noun? Should it just be "more resources"?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamic/DynamicResourceFileActionProvider.kt [hunk 2]
```diff
@@ -11,37 +11,41 @@ import com.intellij.openapi.util.Key
...
                 )
-                false -> DynamicResourceVirtualFilePanel(
-                    project,
-                    file,
-                    message("dynamic_resources.edit_resource_instruction"),
-                    "dynamic.resource.editor.enableEditingResource"
-                )
-            }
-        else -> null
+            is ViewEditableDynamicResourceVirtualFile ->
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamic/DynamicResourceFileActionProvider.kt:32]: this seems unrelated ?

**Review by kiiadi — APPROVED** [review]

File: jetbrains-core/resources/META-INF/plugin.xml (hunk 1 above)
Comment (manodnyab) [reply jetbrains-core/resources/META-INF/plugin.xml]: Changed to create/update/delete

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamic/DynamicResourceFileActionProvider.kt (hunk 2 above)
Comment (manodnyab) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamic/DynamicResourceFileActionProvider.kt:32]: This is the same code but new condition added where the panel(to create/update) is not displayed if the feature is disabled
